test/ui/confirm-test.js

```diff
@@ -17,128 +15,68 @@ for (let key in requirements) {
 // Hyper-V is not enabled on CI
 delete requirements.hyperv;
 
-const messages = {
-  detected: 'Using detected version',
-  older: 'Older than required!',
-  newer: 'Newer than required!'
-};
-
-describe('Confirm page', function confimPage() {
-  let confirmForm, detectionInfo, nextButton;
+describe('Confirmation page', function() {
+  let installsize, downloadSize, backButton, nextButton, cancelButton, browseButton, confirmNote, summaryHeading;
 
   beforeAll(function() {
-    browser.setLocation('confirm')
+    browser.setLocation('selection')
       .then(function() {
-        confirmForm = element(By.id('confirmForm'));
-        detectionInfo = element(By.id('detection-info'));
-        nextButton = element(By.id('confirm-install-btn'));
+        return browser.setLocation('confirmation');
```

Comment:
The URI is just 'confirm' - this makes the tests fail actually :)

Reply:
Updated in the latest commit.